roles/grafana/templates/grafana.ini.j2

```diff
@@ -71,7 +71,8 @@ welcome_email_on_sign_up = {{ grafana_welcome_email_on_sign_up }}
 {% if grafana_auth != {} %}
 [auth]
 disable_login_form = {{ grafana_auth.disable_login_form | default('False') }}
-oauth_auto_login = {{ grafana_auth.oauth_auto_login | default('False') }}
+oauth_allow_insecure_email_lookup = {{ grafana_auth.oauth_allow_insecure_email_lookup | default('False') }}
+disable_signout_menu = {{ grafana_auth.disable_signout_menu | default('False') }}
```

Comment:
Not sure why you are duplicating the line
```
disable_signout_menu = {{ grafana_auth.disable_signout_menu | default('False') }}
```

Also not sure why you are removing 
```
oauth_auto_login = {{ grafana_auth.oauth_auto_login | default('False') }
```

Reply:
sorry, my fault on rebasing, fixed that.

Thanks for the hint